src/libraries/System.Text.Json/tests/Common/ExtensionDataTests.cs

```diff
@@ -520,6 +520,7 @@ public void ExtensionProperty_SupportsWritingToCustomSerializerWithExplicitConve
         [InlineData(typeof(Dictionary<string, JsonElement>), typeof(JsonElementOverflowConverter), typeof(JsonElement))]
         [InlineData(typeof(CustomOverflowDictionary<object>), typeof(CustomObjectDictionaryOverflowConverter), typeof(object))]
         [InlineData(typeof(CustomOverflowDictionary<JsonElement>), typeof(CustomJsonElementDictionaryOverflowConverter), typeof(JsonElement))]
+        [ActiveIssue("https://github.com/dotnet/runtime/issues/86973", typeof(PlatformDetection), nameof(PlatformDetection.IsNativeAot))]
```

Comment:
This doesn't look like a generic theory - what's the issue?

Reply:
The test is running `MethodInfo.MakeGenericInfo()` with the type arguments. Not quite a generic theory but same pattern in principle.

Reply:
We probably just need a line in RD.XML to fix this - the problematic one is the JsonElement. Reference types will Just Work. Valuetypes require runtime codegen.